user-interface/src/main/java/life/qbic/datamanager/views/general/spreadsheet/Spreadsheet.java

```diff
@@ -731,6 +748,13 @@ private void markCellAsInvalid(Cell cell) {
     cell.setCellStyle(invalidCellStyle);
   }
 
+  public void markLastRowInvalid() {
+    int lastRowIndex = rowCount() - 1;
+    Cell sampleCodeCell = getCell(lastRowIndex, 1).get();
```

Comment:
This is not correct. There is nothings stating in the spreadsheet that the first column is the sample code. This is a general spreadsheet not something to do with samples. You need to extract the meaning of the data elsewhere.
Consider SampleBatchInformationSpreadsheet

Reply:
This has been changed to mark a specific cell using the column name and the running number (# column).